DOCS/license-inclusion-principles.md

```diff
@@ -28,7 +28,21 @@ The SPDX legal team will also consider any additional information considered rel
 
 Comments on license inclusion will be noted in the Github issue for the license (or reference to meeting minutes, as necessary). Some license requests may be decided solely via comments in the Github issue and some may involve discussion on the bi-weekly legal call. Thus it is imperative that requestors join the mailing list and calls to fully participate.
 
-*last updated Mar 2020*
```

Comment:
april 2023

Reply:
we put a separate update date for the exception inclusion section. I think having separate updates for each part is good so then it shows with more idea as to what has been updated.
@swinslow - thoughts?